src/components/activities/ActivitySelect.tsx

```diff
@@ -45,6 +47,7 @@ export const ActivitySelect = ({ value, onChange, onSelect, style }: ActivitySel
         villageId: village.id,
         countries: village.countries.map((c) => c.isoCode).join(','),
         pelico: true,
+        type: type,
```

Comment:
If you add `type` in the fetch function, then it's best to put it as well in the dependancy array of the React.useCallback. 
The only problem then, is that if the type change, it will refetch data and have unwanted side-effects. So I would do the following:

```tsx
const previousType = React.useRef(type); // store previous value of type.
const fetchData = React.useCallback(async () => {
    if (!village) {
      return;
    }
    setLoading(true);
    // if type change, reset actions list and page to 0.
    let typeChanged = type !== previousType.current;
    if (type !== previousType.current) {
        dataPage.current = 0;
    }
    [...]
    setActivities((a) => typeChanged ? newActivities : [...a, ...newActivities]);
    [...]
}, [axiosLoggedRequest, village, type]);
```

Reply:
you're right ! i totally forgot